recipe/meta.yaml

```diff
@@ -25,7 +25,7 @@ requirements:
     - dictdiffer >=0.8.1
     - pygtrie >=2.3.2
     - shortuuid >=0.5.0
-    - dvc-objects <2,>=1.0.1
+    - dvc-objects <2,>=1.1.0
```

Comment:
PR at https://github.com/conda-forge/dvc-objects-feedstock/pull/82 is merged already.